### ISSUE TITLE
PermissionError exception on startup with snap installation 

### ISSUE THREAD

**Issue by rstevens011** [Tautulli/Tautulli#1400]
**Describe the Bug**
On a fresh Raspberry Pi with Raspbian 10, I installed snapd for the first time and then installed the latest tautulli (as of this writing v2.6.10).  Upon running `snap start tautulli.tautulli`, I get the following traceback in the snap's log:

```
2021-03-22T20:32:01Z systemd[1]: Started Service for snap application tautulli.tautulli.
2021-03-22T20:32:01Z tautulli.tautulli[1390]: ERROR: ld.so: object '/usr/lib/arm-linux-gnueabihf/libarmmem-${PLATFORM}.so' from /etc/ld.so.preload cannot be preloaded (cannot open shared object file): ignored.
2021-03-22T20:32:01Z tautulli.tautulli[1390]: ERROR: ld.so: object '/usr/lib/arm-linux-gnueabihf/libarmmem-${PLATFORM}.so' from /etc/ld.so.preload cannot be preloaded (cannot open shared object file): ignored.
2021-03-22T20:32:05Z tautulli.tautulli[1390]: ERROR: ld.so: object '/usr/lib/arm-linux-gnueabihf/libarmmem-${PLATFORM}.so' from /etc/ld.so.preload cannot be preloaded (cannot open shared object file): ignored.
2021-03-22T20:32:12Z tautulli.tautulli[1390]: Traceback (most recent call last):
2021-03-22T20:32:12Z tautulli.tautulli[1390]:   File "/snap/tautulli/616/plexpy/logger.py", line 405, in new_run
2021-03-22T20:32:12Z tautulli.tautulli[1390]:     old_run(*args, **kwargs)
2021-03-22T20:32:12Z tautulli.tautulli[1390]:   File "/snap/tautulli/616/usr/lib/python3.6/threading.py", line 864, in run
2021-03-22T20:32:12Z tautulli.tautulli[1390]:     self._target(*self._args, **self._kwargs)
2021-03-22T20:32:12Z tautulli.tautulli[1390]:   File "/snap/tautulli/616/lib/cherrypy/process/servers.py", line 225, in _start_http_thread
2021-03-22T20:32:12Z tautulli.tautulli[1390]:     self.httpserver.start()
2021-03-22T20:32:12Z tautulli.tautulli[1390]:   File "/snap/tautulli/616/lib/cheroot/server.py", line 1795, in start
2021-03-22T20:32:12Z tautulli.tautulli[1390]:     self.prepare()
2021-03-22T20:32:12Z tautulli.tautulli[1390]:   File "/snap/tautulli/616/lib/cheroot/server.py", line 1758, in prepare
2021-03-22T20:32:12Z tautulli.tautulli[1390]:     self.socket.listen(self.request_queue_size)
2021-03-22T20:32:12Z tautulli.tautulli[1390]: PermissionError: [Errno 1] Operation not permitted
2021-03-22T20:32:12Z systemd[1]: snap.tautulli.tautulli.service: Main process exited, code=exited, status=1/FAILURE
2021-03-22T20:32:12Z systemd[1]: snap.tautulli.tautulli.service: Failed with result 'exit-code'.
```

**Steps to Reproduce**

```
sudo apt install snapd
sudo reboot now
...
sudo snap install tautulli
sudo snap start tautulli.tautulli 
# wait 30 seconds to one minute
sudo snap logs -n=all tautulli.tautulli # see output above
```

**Expected Behavior**
The service to start and listen on 0.0.0.0:8181

**Screenshots**
N/A

**Relevant Settings**
It's unclear to me if you have exposed any settings via `snap get...`.  I don't believe you have.

**Tautulli and System Info (see Tautulli settings page)**
Unable to get to web ui to check many of these, but:

```
pi@pi1:~ $ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
```
```
pi@pi1:~ $ snap list
Name      Version   Rev   Tracking  Publisher    Notes
core18    20210128  1989  stable    canonical✓   base
tautulli  v2.6.10   616   stable    jonnywong16  -
```
```
pi@pi1:~ $ snap info tautulli
name:      tautulli
summary:   A Python based monitoring and tracking tool for Plex Media Server.
publisher: Jonathan Wong (jonnywong16)
license:   unset
description: |
  Tautulli is a 3rd party application that you can run alongside your Plex Media Server to monitor
  activity and track various statistics. Most importantly, these statistics include what has been
  watched, who watched it, when and where they watched it, and how it was watched. The only thing
  missing is "why they watched it", but who am I to question your 42 plays of Frozen. All statistics
  are presented in a nice and clean interface with many tables and graphs, which makes it easy to
  brag about your server to everyone else.
services:
  tautulli: simple, enabled, inactive
snap-id:      87nLVs0zl6gEBcvpqfiVppn0sp9uha9W
tracking:     stable
refresh-date: today at 13:02 PDT
channels:
  stable:    v2.6.10 2021-03-17 (616) 31MB -
  candidate: ↑
  beta:      ↑
  edge:      bc47f9a 2021-03-22 (640) 31MB -
installed:   v2.6.10            (616) 31MB -
```

**Link to logs (required)**
Relevant log is above.

**Comment by JonnyWong16** [comment]
I was not able to figure out what permissions are wrong in this Reddit post.

https://reddit.com/r/Tautulli/comments/llg1xc/tautulli_fails_to_start_on_fresh_dietpi_install/

**Comment by samwiseg0** [comment]
Can you try `sudo snap install core`?

https://forum.snapcraft.io/t/installing-snap-on-debian/6742

**Comment by rstevens011** [comment]
I’m surprised this was closed. The README should at least be updated to call out the extra step.

Sent from my iPhone

On Mar 22, 2021, at 6:52 PM, JonnyWong16 ***@***.***> wrote:

﻿

Closed #1400<https://github.com/Tautulli/Tautulli/issues/1400>.

**Comment by samwiseg0** [comment]
Unfortunately, we do not control those install directions. Snap controls them. This is an issue with the raspbian install as other operating systems automatically install core when snapd is installed. This is not an issue with tautulli or the tautulli snap but rather an issue with snapd and how it is installed on raspbian which we have no control over.
